blueprints/app/files/.eslintrc.js

```diff
@@ -23,7 +23,8 @@ module.exports = {
         'index.js',<% } %>
         'testem.js',
         'ember-cli-build.js',
-        'config/**/*.js'<% if (blueprint !== 'app') { %>,
+        'config/**/*.js'<% if (blueprint === 'app') { %>,
+        'lib/**/*.js'<% } %><% if (blueprint !== 'app') { %>,
```

Comment:
It isn't all of `lib/**`, its only `lib/**/index.js` and `lib/**/config/*.js`.

Reply:
oh you're right

Reply:
I went "lib/**/*.js" => "lib/*/index.js". We don't want to match an ember file named "index.js". Also our in-repo addon blueprint doesn't have config, so I left that out.